server/app/app.go

```diff
@@ -283,6 +287,15 @@ func registerConfigResource(configRepo *configresource.Repository, router *mux.R
 	manager.RegisterRoutes(router)
 }
 
+func registerPollingProfilesResource(configRepo *pollingprofile.Repository, router *mux.Router, db *sql.DB) {
+	manager := resourcemanager.New[pollingprofile.PollingProfile](
+		pollingprofile.ResourceName,
+		configRepo,
+		resourcemanager.WithOperations(pollingprofile.Operations...),
+	)
+	manager.RegisterRoutes(router)
+}
```

Comment:
We can rename `configRepo` here to just `repo`:

```suggestion
func registerPollingProfilesResource(repo *pollingprofile.Repository, router *mux.Router, db *sql.DB) {
	manager := resourcemanager.New[pollingprofile.PollingProfile](
		pollingprofile.ResourceName,
		repo,
		resourcemanager.WithOperations(pollingprofile.Operations...),
	)
	manager.RegisterRoutes(router)
}
```